Stylesheets/Components/navigation.css

```diff
@@ -0,0 +1,12 @@
+.demo-wrapper nav{
+    width: 100%
+}
```

Comment:
If this style applies only to the navigation component present in the documentation then you can shift it to the documentation.css file !!